lib/arena.rb

```diff
@@ -1,2 +1,79 @@
+require "pry"
+require_relative "./gladiator"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize name
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator (gladiator)
+    unless gladiators.length == 2
+    gladiators << gladiator
+    end
+  end
+
+  def fight
+    if gladiators.length >= 2
+      if gladiators[0].name == "Maximus"
```

Comment:
Looks like i never pull requested the changes we made on our 1on1!

On Monday, March 14, 2016, Nicholas Olds <notifications@github.com> wrote:

> In lib/arena.rb
> <https://github.com/ga-wdi-exercises/gladiator/pull/51#discussion_r56106019>
> :
>
> > +  attr_accessor :name, :gladiators
> > +
> > +  def initialize name
> > +    @name = name.capitalize
> > +    @gladiators = []
> > +  end
> > +
> > +  def add_gladiator (gladiator)
> > +    unless gladiators.length == 2
> > +    gladiators << gladiator
> > +    end
> > +  end
> > +
> > +  def fight
> > +    if gladiators.length >= 2
> > +      if gladiators[0].name == "Maximus"
>
> what about breaking this series of checks down into helper methods, such
> as maximus? or utilize case statements
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ga-wdi-exercises/gladiator/pull/51/files#r56106019
>